src/globals.d.ts

```diff
@@ -69,3 +69,18 @@ declare class HDWalletProvider {
 declare module 'truffle-hdwallet-provider' {
     export = HDWalletProvider;
 }
+
+// abi-decoder declarations
+interface DecodedLogArg {
+    name: string;
+    value: string|BigNumber.BigNumber;
```

Comment:
Can this really only be string or bignumber? What about boolean?

Reply:
We already had this discussion.
The result of it is `ContractEventArg` type in `types.ts`, which I can't import in `global.ts`.
We have a rule to not define unneccessary types before they're needed.
Currently we will decode only our events and our events have only those types